index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <!-- Nav Bar menu on mobile -->
+  <header>
+    <nav class="mobile-nav-bar">
+      <a href="">My Logo</a>
+      <img src="Assets/Unionhamb.png" alt="mobile menu">
+    </nav>
+  </header>
+
+  <!-- Section Introduction and -->
+
+  <section class="intro-section">
+    <h1 class="intro-section__title">I’m Arthur. 
+    Glad to see you!</h1>
+    <h3 class="intro-section__subtitle">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
+    <h3 class="intro-section__capstext">LET’S CONNECT</p>
```

Comment:
- [ ] I'm sorry for the mixup 🙄, the previous reviewer was saying that it is good to use hx (examples: h2, h3, h4, h5, h6)  for the subtitle i.e. `LET’S CONNECT`

- [ ] Please kindly make the paragraph (e.g., "I'm a software ...") a `p` tag.

- [ ] This way will be semantically correct and the screen readers and other technologies will thank you. 🤗

- [ ] Also I noticed that you have closed the `<h3>` tag with `<p>` tag. please kindly fix that

Your code 👇 
```HTML
<h3 class="intro-section__capstext">LET’S CONNECT</p>
```

Reply:
Thank you @abdulhamiid for clarifying that.